apisupport/Build/apisupport.sh

```diff
@@ -0,0 +1,54 @@
+# Should be sourced by `command -p sh path/to/apisupport.sh "$PATH"` from
+# within a Makefile.
+if ! [ ${PATH} = "$1" ]; then
+	echo "WARNING: POSIX violation: $SHELL's command -p resets \$PATH" 1>&2
+	PATH=$1
+fi
+# Standard output should be written to apisupport-config.h, which is both a
+# C header file defining APISUPPORT_ARCH_FEATURE macros and sourceable sh
+# code which sets CFLAGS_ARCH_FEATURE environment variables.
+SRCDIR=$(command -p dirname "$0")
+
+CFLAGS_HARDCODED="-D_POSIX_C_SOURCE=200809L -D_XOPEN_SOURCE=700"
+
+feature() {
+	ARCH=$1
```

Comment:
s/ARCH/PLATFORM/ ?

Reply:
Hmm, good point.  I'm happy calling it PLATFORM, but at the moment, I'm using that variable to be `NONPOSIX` and `LIBSSL`.  Is that stretching the meaning of "platform"?

Reply:
Maybe a bit of a stretch but it's better than anything else which comes to mind.

Reply:
ok, done as a REBASE commit.